README.md

```diff
@@ -21,70 +21,74 @@ npm i zos-lib
 Next, learn how to:
 - [Develop and deploy a single smart contract which can be upgraded](#single) (for bugfixing or adding new features).
 - [Develop and operate a complex upgradeable app](#complex) with multiple smart contracts which are connected to the zOS Kernel upgradeable standard libraries.
-- [Develop a zOS Kernel standard library release.](https://github.com/zeppelinos/kernel#developing-kernel-standard-libraries)
```

Comment:
why remove this?

Reply:
Because this links to another document, I was expecting to find it after the complex example. I moved it to the bottom, after a standard library is used. At the end of the README we can put a section to "read more", this link should go there IMO.